### PR TITLE
Logging and other small warnings removal

### PR DESCRIPTION
**Pull request by tbenr**
I don't know if everything is correct here, I'm not a kotlin expert

### CODE REVIEW

**Comment by tbenr** [comment]
@Nashatyrev @StefanBratanov

File: libp2p/src/main/kotlin/io/libp2p/pubsub/AbstractRouter.kt [hunk 1]
```diff
@@ -23,7 +23,7 @@ const val DEFAULT_MAX_PUBSUB_MESSAGE_SIZE = 1 shl 20
 
 typealias PubsubMessageHandler = (PubsubMessage) -> CompletableFuture<ValidationResult>
 
-open class DefaultPubsubMessage(override val protobufMessage: Rpc.Message) : AbstractPubsubMessage() {
+class DefaultPubsubMessage(override val protobufMessage: Rpc.Message) : AbstractPubsubMessage() {
```
Comment (Nashatyrev) [inline libp2p/src/main/kotlin/io/libp2p/pubsub/AbstractRouter.kt:26]: Until this project has strict rules regarding members visibility I would suggest to leave them as is. I was using some of them for tweaking router behavior in side projects.

File: libp2p/src/main/kotlin/io/libp2p/pubsub/AbstractRouter.kt [hunk 2]
```diff
@@ -105,7 +104,7 @@ abstract class AbstractRouter(
     }
 
     override fun addNewStream(stream: Stream) = addNewStreamWithHandler(stream, null)
-    protected fun addNewStreamWithHandler(stream: Stream, handler: ChannelHandler?) {
+    private fun addNewStreamWithHandler(stream: Stream, handler: ChannelHandler?) {
```
Comment (Nashatyrev) [inline libp2p/src/main/kotlin/io/libp2p/pubsub/AbstractRouter.kt:107]: Same here

**Review by Nashatyrev — APPROVED** [review]
LGTM, but I would ask to keep the stuff open for hacking

**Pull request merged**